critiquebrainz/frontend/templates/review/modify/event.html

```diff
@@ -11,7 +11,11 @@
       {% endif %}
     </dd>
     <dt>{{ _('Place') }}</dt>
-    <dd>{{ entity['place-relation-list'][0]['place']['name'] or '-' }}</dd>
+    {% if entity['place-relation-list'] is not defined %}
+      <dd>{{ _('Unknown place') }}</dd>
```

Comment:
Can you make this stand out from styling of regular name somehow? Maybe add `text-muted` class?